### DISCUSSION TITLE
Buy and place furniture in campaign

### DISCUSSION THREAD

**Discussion by RadCatish** [FakeFishGames/Barotrauma#4809] [Ideas and suggestions]
Might it be possible to buy new furniture (cabinets, crate racks, diving suit lockers etc) at stations and place them during the campaign? I think it would be really satisfying to build a better and more well-stocked sub as you go along

**Comment by Kulik242** [comment]
It would be great, but I imagine hard to implement. Maybe one day...

**Reply by mLuby** [reply]
It's been done by many mods already, so actually quite easy.
